Storage/src/Connection/Rest.php

```diff
@@ -49,23 +49,27 @@ class Rest implements ConnectionInterface
     /**
      * @deprecated
      */
-    const BASE_URI = 'https://storage.googleapis.com/storage/v1/';
+    public const BASE_URI = 'https://storage.googleapis.com/storage/v1/';
 
-    const DEFAULT_API_ENDPOINT = 'https://storage.googleapis.com';
+    public const DEFAULT_API_ENDPOINT = 'https://storage.googleapis.com';
 
     /**
      * @deprecated
      */
-    const UPLOAD_URI = 'https://storage.googleapis.com/upload/storage/v1/b/{bucket}/o{?query*}';
+    public const UPLOAD_URI = 'https://storage.googleapis.com/upload/storage/v1/b/{bucket}/o{?query*}';
```

Comment:
The linter changes const to public const. Though it would not break any functionality, but it does breaks uniformity, So reverted it for now.

Do we need to add them?

Reply:
Reverted them